src/sections/Home/Proud-maintainers/proudMaintainers.style.js

```diff
@@ -5,8 +5,18 @@ const ProjectItemWrapper = styled.section`
 		align-self: flex-start;
 		margin-left: -5rem;
 		padding: 0rem;
+		@media screen and (max-width: 1600px) {
+			margin-left: -12rem;
+		}
 		@media screen and (max-width: 992px) {
-			margin-left: 0rem;
+			margin-left: -10rem;
+		}
+		@media screen and (max-width: 768px) {
+			margin-left: -8rem;
```

Comment:
this gets us offscreen

![image](https://user-images.githubusercontent.com/95926324/186675160-63d06c9e-ea2f-4226-99a9-90b53d9f4514.png)